File: nutrinova-api/NutrinovaData/Features/Reports/PatientNutrientGoalReport.cs
namespace NutrinovaData.Features.Reports;

public class PatientNutrientGoalReport
{
  public string PatientName { get; set; } = string.Empty;

  public DateTime ReportBegin { get; set; }

  public DateTime ReportEnd { get; set; }

  public IEnumerable<NutrientGoalReportItem> NutrientGoalReportItems { get; set; } = new List<NutrientGoalReportItem>();
}

File: nutrinova-ui/src/app/(authorized)/goals/_models/PatientNutrientGoalReport.ts
import { NutrientGoalReportItem } from "./NutrientGoalReportItem";

export interface PatientNutrientGoalReport {
  patientId: string;
  patientName: string;
  reportBegin: Date;
  reportEnd: Date;
  nutrientGoalReportItems: NutrientGoalReportItem[];
}
